File: gdscript/src/main/java/gdscript/psi/GdFlowSt.java
package gdscript.psi;

import org.jetbrains.annotations.*;

public interface GdFlowSt extends GdStmt {

  @NotNull
  GdEndStmt getEndStmt();

  @Nullable
  GdExpr getExpr();

  @NotNull
  String getType();

}

File: gdscript/src/main/java/gdscript/psi/GdFlowSt.kt
package gdscript.psi;

import org.jetbrains.annotations.NotNull;
import org.jetbrains.annotations.Nullable;

public interface GdFlowSt extends GdStmt {

  @NotNull
  GdEndStmt getEndStmt();

  @Nullable
  GdExpr getExpr();

  @NotNull
  String getType();

}
